pymongo/common.py

```diff
@@ -169,6 +171,12 @@ def raise_config_error(key: str, dummy: Any) -> NoReturn:
 }
 
 
+def camel_to_snake(camel):
```

Comment:
Can we somehow combine this instance of this method with the one in `test/utils.py`?

Reply:
Done